WordPress/Sniffs/VIP/SuperGlobalInputUsageSniff.php

```diff
@@ -43,6 +66,24 @@ public function register() {
 	 * @return void
 	 */
 	public function process_token( $stackPtr ) {
+		if ( false === $this->thrown['DeprecatedSniff'] ) {
+			$this->thrown['DeprecatedSniff'] = $this->phpcsFile->addWarning(
+				'The "WordPress.VIP.SessionVariableUsageSniff" sniff has been deprecated. Please update your custom ruleset.',
```

Comment:
References the wrong sniff: `WordPress.VIP.SessionVariableUsageSniff` should be `WordPress.VIP.SuperGlobalInputUsage`.